seq2science/rules/quantification.smk

```diff
@@ -142,99 +142,39 @@ if config["quantifier"] == "salmon":
             salmon quant -i {input.index} -l A {params.input} {params.params} -o {output.dir} \
             --threads {threads} > {log} 2>&1
             """
+            
+            
+elif  "scrna_seq" == get_workflow():
 
-
-elif config["quantifier"] == "kallistobus":
-    if 'kite' in config.get('ref',""):
-        ruleorder: kallistobus_ref_kite > get_genome
-        ruleorder: kallistobus_ref_kite > kallistobus_ref
-    else:
-        ruleorder: kallistobus_ref > kallistobus_ref_kite
-        
-        
-    rule kallistobus_ref:
-        """
-        Make a genome index for kallistobus. This index is required for counting.
-        """
-        input:
-            fa=expand("{genome_dir}/{{assembly}}/{{assembly}}.fa", **config),
-            gtf=expand("{genome_dir}/{{assembly}}/{{assembly}}.annotation.gtf", **config),
-        output:
-            directory(expand("{genome_dir}/{{assembly}}/index/kallistobus/", **config)),
-        log:
-            expand("{log_dir}/kallistobus_index/{{assembly}}.log", **config),
-        benchmark:
-            expand("{benchmark_dir}/kallistobus_index/{{assembly}}.benchmark.txt", **config)[0]
-        priority: 1
-        conda:
-            "../envs/kallistobus.yaml"
-        resources:
-            mem_gb=88,
-        params:
-            basename=lambda wildcards, output: f"{output[0]}{wildcards.assembly}",
-            options=config.get("ref")
-        shell:
-            """
-            kb ref \
-            {input.fa} {input.gtf} \
-            -i {params.basename}.idx -g {params.basename}_t2g.txt -f1 {params.basename}_cdna.fa \
-            -f2 {params.basename}_intron.fa \
-            -c1 {params.basename}_cdna_t2c.txt -c2 {params.basename}_intron_t2c.txt \
-            {params.options} > {log} 2>&1
-            """
-                
-                
-    rule kallistobus_ref_kite:
-        """
-        Make a mismatch index for kallistobus. This index is required to count feature barcodes, such as antibody tags. 
-        """
-        input:
-            featurebarcodes=expand("{genome_dir}/{{assembly}}.tsv", **config)
-        output:
-            directory(expand("{genome_dir}/{{assembly}}/index/kallistobus/kite/", **config)),
-        log:    
-            expand("{log_dir}/kallistobus_index_kite/{{assembly}}.log", **config),    
-        conda:
-            "../envs/kallistobus.yaml"  
-        resources:
-            mem_gb=12, 
-        params:    
-            basename=lambda wildcards, output: f"{output[0]}{wildcards.assembly}",
-            options=config.get("ref")
-        priority: 1
-        shell:
-            """
-            mkdir -p {params.basename}
-            kb ref  \
-            {input.featurebarcodes} \
-            {params.options} \
-            -i {params.basename}.idx -g {params.basename}_t2g.txt -f1 {params.basename}_cdna.fa > {log} 2>&1 
-            """    
-        
-        
     def get_fastq_pair_reads(wildcards):
+        
```

Comment:
:innocent:

Reply:
This might *actually be important* for the automatic parsing in the docs :stuck_out_tongue: